src/components/tracked-controls.js

```diff
@@ -18,7 +19,10 @@ module.exports.Component = registerComponent('tracked-controls', {
     controller: {default: 0},
     id: {type: 'string', default: ''},
     idPrefix: {type: 'string', default: ''},
-    rotationOffset: {default: 0}
+    rotationOffset: {default: 0},
+    // Arm model parameters, to use when not 6DOF. (pose hasPosition false, no position)
+    headElement: {type: 'selector'},
+    defaultHand: {type: 'string', default: 'right'}
```

Comment:
We can also hide this as a constant. I've talked to someone working with the daydream SDK and you can get the handedness configured by the user. The info should be passed to the browser via the `hand` attribute on the gamepad object similarly to the Oculus controllers. It should not be an application configurable property.